app/logic/volunteers.py

```diff
@@ -92,3 +97,34 @@ def setUserBackgroundCheck(user,bgType, checkPassed, datePassed):
     update = BackgroundCheck.create(user=user, type=bgType, passBackgroundCheck=checkPassed, datePassed=datePassed)
     update.save()
     createLog(f"Updated {user.firstName} {user.lastName}'s background check for {bgType} to {bool(checkPassed)}.")
+
+def getProgramManagerForEvent(user, event= None, programId = None):
```

Comment:
Two issues here:

1. The name is a bit off. It makes me think this is a function for figuring out who is the program manager for an event. If it returns a boolean, it should be something like `isProgramManagerForEvent`.
2. We already have a function to check if a user has permissions to mess with the program, `hasPrivilege` (also not a great name, admittedly). this one should just take a user and event, and call hasPrivilege to check the program.